### PR TITLE
Tweak Android versionCode to follow git tag

### PR DESCRIPTION
**Pull request by AndrewGable**
### Details
Adds logic back to follow git tag with Android Version code. The final syntax we agreed on was:

* `PREFIX` (required by Android because of our previous versions): `10`
* `MAJOR`: 2 digits - Minimum set to `01` currently
* `MINOR`: 2 digits - Minimum set to `00` currently
* `PATCH`: 2 digits - Minimum set to `02` currently
* `BUILD`: 2 digits - Set to ~`15` currently

### Fixed Issues
Fixes https://github.com/Expensify/Expensify/issues/156873

### Tests
1. Jest tests 👯‍♂️

### CODE REVIEW

File: .github/libs/versionUpdater.js [hunk 1]
```diff
@@ -24,12 +24,15 @@ const getVersionNumberFromString = (versionString) => {
  * @param {Number} major
  * @param {Number} minor
  * @param {Number} patch
- * @param {Number} [build]
+ * @param {Number} build
```
Comment (roryabraham) [inline .github/libs/versionUpdater.js:27]: I think this is still an optional parameter, so it should have a default and square brackets in the JSDoc to mark it as optional:

```javascript
/**
 * ...
 * @param {Number} [build]
 * @returns {String}
 */
const getVersionStringFromNumber = (major, minor, patch, build = 0) => {
    return `${major}.${minor}.${patch}-${build}`;
}
```

**Review by roryabraham — CHANGES_REQUESTED** [review]
Just one minor request

**Comment by roryabraham** [comment]
Also, merge conflicts

**Review by roryabraham — APPROVED** [review]
LGTM 👍

**Review by TomatoToaster — APPROVED** [review]
Lol I love reading these almost english tests LGTM Already merged, but it didn't say so while reviewing ignore this!

**Pull request merged**